examples/seismic/viscoelastic/operators.py

```diff
@@ -49,7 +49,7 @@ def ForwardOperator(model, geometry, space_order=4, save=False, **kwargs):
     u_v = Eq(v.forward, damp * (v + s*ro*div(tau)))
     symm_grad = grad(v.forward) + grad(v.forward).T
     # Stress equations:
-    u_t = Eq(tau.forward, damp * (r.forward + tau +
+    u_t = Eq(tau.forward, damp * (s*r.forward + tau +
```

Comment:
You may need to change the norm to test against in `examples/seismic/viscoelastic/viscoelastic_example.py`

Reply:
Ok. Sorry!
 I wiil do that.

Reply:
I've already made the changes in `examples/seismic/viscoelastic/viscoelastic_example.py`